### PR TITLE
Enable maintainer to manually approve..

### PR DESCRIPTION
**Pull request by sbose78**
operator upgrades

Typically, if the operator upgrade strategy is set to 'manual', when a new version is available for approval - 
* User would need to find the relevant installPlan
* Edit the InstallPlan and set "approved" to `true`.
```
spec:
    approval: Manual
    approved: false # flip to true
```

OpenShift Consoles makes this less error prone but users/maintainers still need permissions on the InstallPlan resource to be able to do this.

### CODE REVIEW

File: components/authentication/base/pipeline-service-sre.yaml [hunk 1]
```diff
@@ -3,6 +3,14 @@ apiVersion: rbac.authorization.k8s.io/v1
...
   name: pipeline-service-sre
 rules:
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - installplans
+  verbs:
+  - get
+  - list
+  - patch
```
Comment (adambkaplan) [inline components/authentication/base/pipeline-service-sre.yaml:13]: No update?

**Comment by sbose78** [comment]
Adam, Ann,
Reviews addresses.

**Review by amfred — APPROVED** [review]

**Pull request merged**